kustomize/overlays/ci/shard/event-bridge-shard-operator-secrets.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: addon-smart-events-operator-parameters
```

Comment:
As you moved to from `event-bridge-shard-operator-secrets` to `addon-smart-events-operator-parameters` wdyt about renaming also the file so to match the name of the resource?

Reply:
I keep it with same name just to allign with other config file.  `event-bridge-shard-operator-config.yaml`

Reply:
Yes, I see. I'm wondering if we should keep the name of the resources aligned with the filename. Is there a specific reason to keep it not aligned?

Reply:
I had rename the file to `addon-smart-events-operator-parameters.yaml`